tests/compareUpscaling.cpp

```diff
@@ -49,20 +49,20 @@ void startTest(const char* testFile, const char* compareFile,  double absoluteTo
     std::vector<double> testVec, compareVec;
     readData(testFile, testVec);
     readData(compareFile, compareVec);
-  
+
     if(testVec.size()!= compareVec.size()){
 	OPM_THROW(std::invalid_argument, "The upscaling files do not have the same number of values.");
     }
-  
-    double absDev, relDev; 
+
+    double absDev, relDev;
     for (size_t i = 0; i < testVec.size(); i++){
 
-	absDev = std::abs(testVec[i]-compareVec[i]); 
+	absDev = std::abs(testVec[i]-compareVec[i]);
 	relDev = absDev/double(std::max(std::abs(testVec[i]),std::abs(compareVec[i])));
 	
 	if (relDev > relativeTolerance && absDev > absoluteTolerance){
-	    for(size_t i = 0; i < testVec.size(); i++){
-		std::cout << "For data value " << i << "- The relative deviation is "<< relDev << " greater than the limit "<< relativeTolerance << std::endl; 
+	    for(auto val : testVec) {
+		std::cout << "For data value " << val << "- The relative deviation is "<< relDev << " greater than the limit "<< relativeTolerance << std::endl;
```

Comment:
This changes the meaning of the output, but I think it is ok.

Reply:
that is unfortunate, was not my intention to change it, but i guess it's okay.